src/core/requestService/IdentityRequests.ts

```diff
@@ -1,7 +1,11 @@
 import OneSignalApiBaseResponse from "../../shared/api/OneSignalApiBaseResponse";
 import { logMethodCall } from "../../shared/utils/utils";
 import OneSignalError from "../../shared/errors/OneSignalError";
-import ExecutorResult from "../executors/ExecutorResult";
+import ExecutorResult, {
```

Comment:
unneeded import

Reply:
It's still being used as below as as type see this line:
```typescript
static async addIdentity<Model>(operation: Operation<Model>): Promise<ExecutorResult<IdentityModel>> {
```